libs/dh/metering-point/feature-overview/src/lib/secondary-master-data/dh-secondary-master-data.component.html

```diff
@@ -212,12 +180,8 @@
           {{ t('ratedCurrent') }}
         </dt>
         <dd class="watt-text-m">
-          <ng-container
-            *ngIf="secondaryMasterData?.ratedCurrent; else emptyField"
-          >
-            {{ secondaryMasterData?.ratedCurrent }}
-            {{ t('units.ampere') }}
-          </ng-container>
+          {{ secondaryMasterData?.ratedCurrent | emDashFallback }}
```

Comment:
Question: wouldn't this change cause ampere to be shown even though ratedCurrent is not defined?

Reply:
Yes. good catch. I think the options are similar to my previous comment above (under `capacity`).